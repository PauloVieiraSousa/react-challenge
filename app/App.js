
import React from 'react';
import ReactDOM from 'react-dom';
import Home from './components/Home';

ReactDOM.render(<Home />, document.getElementById('app'));

/*
var React = require('react');
var ReactDOM = require('react-dom');


ReactDOM.render(<h1>Ol√° Mundo</h1>, document.getElementById('app'));

*/