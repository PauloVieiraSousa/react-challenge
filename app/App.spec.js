import React from 'react';
import renderer from 'react-test-renderer';
import App from './App'
import 'jest-styled-components';

test ('example test with jest', ()=> {
    const component = renderer.create(<App />).toJSON();
    
    expect(component).toMatchSnapshot();
});

