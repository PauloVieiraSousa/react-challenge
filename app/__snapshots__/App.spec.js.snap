// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example test with jest 1`] = `
<h1>
  Exemplo App
</h1>
`;
